### PR TITLE
Updating the readme doc

### LINKED ISSUE #56: Large Compilations
I am following the directions for building the runtime and I can produce the executables however the runtime executable and the wasm file are huge. 

 372525848 Jan  3 13:16 js-compute-runtime
353101412 Jan  3 13:12 js-compute-runtime.wasm

I tried a few different times but I must be missing something, is there a trick to compile it smaller?

### PR DESCRIPTION
**Pull request by black2d**
Updating the readme to make it easier for new developers to compile the library.

An update to PR #57 and fixes #56.

### CODE REVIEW

**Comment by black2d** [comment]
@tschneidereit any update on merging this?

File: README.md [hunk 1]
```diff
@@ -21,13 +21,63 @@ To build from source, you need to ensure that the headers and object files for t
...
+- cbindgen
+  ```sh
+  cargo install cbindgen
+  ```
+- wasi-sdk
+  ```sh
+  curl -sS -L -O https://github.com/WebAssembly/wasi-sdk/releases/download/wasi-sdk-12/wasi-sdk-12.0-linux.tar.gz
+  tar xf wasi-sdk-12.0-linux.tar.gz
+  sudo mkdir -p /opt/wasi-sdk
+  sudo mv wasi-sdk-12.0/* /opt/wasi-sdk/
```
Comment (elliottt) [inline README.md:55]: Is it worth pointing out that there's a `.deb` available for ubuntu/debian users?

**Review by elliottt — APPROVED** [review]
These build directions are great, thank you for putting them together!

I only had one comment about debian packages being available, feel free to ignore it :)

**Comment by black2d** [comment]
@elliottt no worries I wrote them a few months back. I may take another pass on a fresh load and make sure nothing has changed.

**Pull request merged**